### ISSUE TITLE
Path syntax conversion when pasting pathname

### ISSUE THREAD

**Issue by waghcwb** [mintty/mintty#975]
It is a common task to copy an path from Windows Explorer and then `cd` in that folder using the terminal. But when I copy an path from Windows Explorer it comes with backslashes (eg: `C:\Users\your-awesome-user\Documents`), so I need to manually convert to this `C:/Users/your-awesome-user/Documents` before I can `cd` in that folder.

**Comment by mintty** [comment]
Duplicate of #785 and #644. Use drag-and-drop from Explorer to invoke path syntax conversion.

**Comment by mintty** [comment]
There are shortcuts Shift+Insert and [Shift+]middle-mouse-click for pasting.
Actually, I could imagine to add variants Alt+Shift+Insert and Alt+[Shift+]middle-click to enforce path conversion.

**Comment by mintty** [comment]
Also, you can already configure this function to be available for keys or in the context menu:
```
KeyFunctions=AS+Insert:paste-path;AS+KP_Insert:paste-path
```
```
CtxMenuFunctions=Paste path:paste-path
MenuMouse=bu
```

**Comment by mintty** [comment]
Added example above to the manual page.

**Comment by mintty** [comment]
Released 3.4.0.
